src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3889,6 +3896,92 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
+        {
+            Debug.Assert(_currentStatement == operation);
+
+            var parameterRef = new ParameterReferenceExpression(operation.Parameter, semanticModel: null,
+                operation.Syntax, operation.Parameter.Type, constantValue: default, isImplicit: true);
+            VisitInitializer(rewrittenTarget: parameterRef, initializer: operation);
+            return null;
+        }
+
+        public override IOperation VisitFieldInitializer(IFieldInitializerOperation operation, int? captureIdForResult)
+        {
+            Debug.Assert(_currentStatement == operation);
+
+            foreach (IFieldSymbol fieldSymbol in operation.InitializedFields)
+            {
+                IInstanceReferenceOperation instance = fieldSymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, fieldSymbol.ContainingType, constantValue: default, isImplicit: true);
+                var fieldRef = new FieldReferenceExpression(fieldSymbol, isDeclaration: false, instance, semanticModel: null,
+                    operation.Syntax, fieldSymbol.Type, constantValue: default, isImplicit: true);
+                VisitInitializer(rewrittenTarget: fieldRef, initializer: operation);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitPropertyInitializer(IPropertyInitializerOperation operation, int? captureIdForResult)
+        {
+            Debug.Assert(_currentStatement == operation);
+
+            foreach (IPropertySymbol propertySymbol in operation.InitializedProperties)
+            {
+                var instance = propertySymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, propertySymbol.ContainingType, constantValue: default, isImplicit: true);
+
+                ImmutableArray<IArgumentOperation> arguments;
+                if (!propertySymbol.Parameters.IsEmpty)
+                {
+                    // Must be an error case of initializing a property with parameters.
+                    var builder = ArrayBuilder<IArgumentOperation>.GetInstance(propertySymbol.Parameters.Length);
+                    foreach (var parameter in propertySymbol.Parameters)
+                    {
+                        var value = new InvalidOperation(ImmutableArray<IOperation>.Empty, semanticModel: null,
+                            operation.Syntax, parameter.Type, constantValue: default, isImplicit: true);
+                        var argument = new ArgumentOperation(value, ArgumentKind.Explicit, parameter, inConversionOpt: null,
+                            outConversionOpt: null, semanticModel: null, operation.Syntax, isImplicit: true);
```

Comment:
>null [](start = 46, length = 4)

Consider using ConvertibleConversion.Instance instead

Reply:
> Consider using ConvertibleConversion.Instance instead

Actually, looking at the baseline, passing null works great as well.  

---
In reply to: [187743952](https://github.com/dotnet/roslyn/pull/26763#discussion_r187743952) [](ancestors = 187743952)